Cargo.toml

```diff
@@ -102,11 +112,9 @@ default = [
     "rustls",
     "rustls-pemfile",
     "inquire",
+    "unsafe-js"
```

Comment:
I've added `unsafe-js` feature to default set to make sure all ci checks work with it.